examples/js/loaders/GLTFLoader.js

```diff
@@ -1341,43 +1342,19 @@ THREE.GLTFLoader = ( function () {
 		this._withDependencies( [
 
 			'scenes',
-			'cameras',
 			'animations'
 
 		] ).then( function ( dependencies ) {
 
-			var scenes = [];
-
-			for ( var i = 0; i < dependencies.scenes.length; i ++ ) {
-
-				scenes.push( dependencies.scenes[ i ] );
-
-			}
-
-			var scene = json.scene !== undefined ? dependencies.scenes[ json.scene ] : scenes[ 0 ];
-
-			var cameras = [];
-
-			dependencies.cameras = dependencies.cameras || [];
+			var scenes = dependencies.scenes;
+			var scene = scenes[ json.scene || 0 ];
+			var animations = dependencies.animations;
```

Comment:
Is this intended change that `animations` will be `undefined` if glTF doesn't have animations while it was `[]`?